web/src/client/questions.js

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+const DBUS_CONFIG = {
+  questions: {
+    path: "/org/opensuse/DInstaller/Questions1",
+    ifaces: {
+      objectManager: "org.freedesktop.DBus.ObjectManager"
+    }
+  },
+  question: {
+    ifaces: {
+      question: "org.opensuse.DInstaller.Question1",
+      luksActivation: "org.opensuse.DInstaller.Question.LuksActivation1"
+    }
+  }
+};
+
+const QUESTION_TYPES = {
+  generic: "generic",
+  luksActivation: "luksActivation"
+};
+
+/**
+ * Builds a question from the given D-Bus question
+ *
+ * @param {Object} dbusQuestion
+ * @returns {Object}
+*/
+function buildQuestion(dbusQuestion) {
+  const getIfacesAndProperties = (dbusQuestion) => Object.values(dbusQuestion)[0];
```

Comment:
code, blank, code, blank... :-) I do not see the point at all. I would get it if the function definition took several lines, but that's not the case. But it is just my POV.

Reply:
I see your point. What is more, I'd move those functions outside the buildQuestion... a matter of taste. Let's remove the blank lines by now.